js/ResourcesView.js

```diff
@@ -12,6 +12,8 @@ export default class ResourcesView extends Backbone.View {
   initialize() {
     this.listenTo(Adapt, 'remove', this.remove);
     this.render();
+
+    this.onResourcesItemClicked = this.onResourcesItemClicked.bind(this);
```

Comment:
These should occur befor everything else in the initialise function, as often function binds are used in listenTo and render functions, as in this case.

Reply:
Done! Thank you.